lib/launcher.js

```diff
@@ -92,7 +92,7 @@ class AppiumLauncher {
         });
     }
 
-    _getAppiumFileName(){
+    _getAppiumFileName (): string {
```

Comment:
Doesn't Flow infer the return type from the function's return expression?

Reply:
Yes there was exception that function does not have return type

Reply:
I got it.